Gemfile

```diff
@@ -5,7 +5,7 @@ gemspec
 gem "minitest", ">= 5"
 gem "rake"
 
-gem "activerecord", "~> 7.0.0"
+gem "activerecord"
```

Comment:
Ensures we run local tests on the latest Rails. CI should be responsible for catching backward incompatible changes.